### ISSUE TITLE
Internal ID mapping in case of custom idAccessor

### ISSUE THREAD

**Issue by aliaksandr-yanchuk** [brimdata/react-arborist#89]
Hi and thanks for a great tree component, really awesome!

I've noticed one thing, when in case of custom idAccessor (e.g. uid instead of id), internal logic is still relying on "id" in some places. For example drag and drop reorder/move works through useSimpleTree and there goes direct mapping through id, without check for idAccessor property.

Best regards,
Alexander Yanchuk

**Comment by jameskerr** [comment]
You are correct. That is a gap that needs closing. useSimpleTree should accept the accessor arguments. The useSimpleTree hook is not really documented as public API, but several people have found it and found this limitation. I'd welcome a PR if you're up for it. It shouldn't be too difficult.

**Comment by aliaksandr-yanchuk** [comment]
Thanks, schedule is a bit tight now, but I'll try

**Comment by TrevorBurnham** [comment]
Consolidating into #347, which tracks this. Closing here to keep the discussion in one place — please follow / weigh in on #347.
